### PR TITLE
BOM-1449 Updated to support Django2

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-1449](https://openedx.atlassian.net/browse/BOM-1449)

### Description
- Added support for Django111, 20, 21 & 22
- Added support for Python3.5
- Removed support for Python2.7
- Removed tests for Python3.6

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,27 +3,19 @@ addons:
     packages:
     - nodejs
 language: python
+python:
+  - 3.5
+  - 3.6
```
Comment (jmbowman) [inline .travis.yml:8]: Shouldn't need the 3.6 tests.

File: setup.py [hunk 2]
```diff
@@ -54,7 +54,11 @@
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: JavaScript',
-        'Programming Language :: Python',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Framework :: Django :: 2.0',
```
Comment (jmbowman) [inline setup.py:59]: 1.11 also?

File: tox.ini [hunk 3]
```diff
@@ -1,11 +1,5 @@
 [tox]
-envlist =
-    csslint
-    eslint
-    py27
-    py36
-    pycodestyle
-    pylint
+envlist = csslint,eslint,pycodestyle,pylint,py{35,36}-django{20,21,22}
```
Comment (jmbowman) [inline tox.ini:2]: Should be testing 1.11 until we're actually off it.

File: .travis.yml (hunk 1 above)
Comment (UsamaSadiq) [reply .travis.yml]: Removed 3.6 tests.

File: setup.py (hunk 2 above)
Comment (UsamaSadiq) [reply setup.py]: Added django111 environment support.

File: tox.ini (hunk 3 above)
Comment (UsamaSadiq) [reply tox.ini]: Added 1.11 testing support.

**Pull request merged**